lib/model/OpenFoodAPISettings.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:openfoodfacts/utils/QueryType.dart';
+
+class OpenFoodAPISettings {
+  static OpenFoodAPISettings? instance;
```

Comment:
Right now we don't need this static field, as you use `OpenFoodAPISettings.fromQueryType(queryType)` in `openfoodfacts.dart`.